docs/glossary/entry-name.md

```diff
@@ -0,0 +1,3 @@
+# Entry Name
+
+Entry name is a property of an entry. Rolldown will use the entry name to generate the file name for the corresponding entry chunk, which will replace the `[name]` placeholder in the [`output.entryFileNames`](../options/output#entryfilenames) pattern.
```

Comment:
Should we link to the entry chunk config here?

Reply:
Yes. It's an unfinished puzzle a lot of to done. Could wait for a time we could batch the changes.